### PR TITLE
Fix grammar in the Golf Code challenge directions

### PR DESCRIPTION
**Pull request by jamigibbs**
Correcting the grammar in the [Golf Code](http://www.freecodecamp.com/challenges/golf-code) challenge.

Tested on local setup. No issue to reference.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 1]
```diff
@@ -2970,8 +2970,8 @@
       "id": "5664820f61c48e80c9fa476c",
       "title": "Golf Code",
       "description": [
-        "In the game of <a href='https://en.wikipedia.org/wiki/Golf' target='_blank'>golf</a> each hole has a <dfn>par</dfn> for the average number of <dfn>strokes</dfn> needed to sink the ball. Depending on how far above or below <code>par</code> your <code>strokes</code> are, there is a different nickname.",
-        "Your function will be passed a <code>par</code> and <code>strokes</code>. Return strings according to this table (based on order of priority - top (highest) to bottom (lowest)):",
+        "In the game of <a href=\"https://en.wikipedia.org/wiki/Golf\" target=\"_blank\">golf</a> each hole has a <code>par</code> for the average number of <code>strokes</code> needed to sink the ball. Depending on how far above or below <code>par</code> your <code>strokes</code> are, there is a different nickname.",
+        "Your function should pass <code>par</code> and <code>strokes</code> arguments and return the correct string according to this table based on the order of priority; top (highest) to bottom (lowest):",
```
Comment (ltegman) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: `should pass` should be something like `should accept` or `will be passed`.

Comment (QuincyLarson) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: @jamigibbs I agree with @ltegman - this should be passive voice.

**Comment by SaintPeter** [comment]
@jamigibbs Will you be fixing this one?

**Comment by jamigibbs** [comment]
@SaintPeter Yep! I'll carve out some time for this today. :)

**Comment by ltegman** [comment]
@jamigibbs Looks good now. :+1:  Just [squash your commits](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) and this will be ready to merge :smile:

**Comment by jamigibbs** [comment]
Sorry about! Should be good to go now. :sunglasses:

**Comment by SaintPeter** [comment]
:+1:

**Pull request merged**